internal/db/rdb/mysql/dbr/dbr_mock.go

```diff
@@ -0,0 +1,126 @@
+package dbr
```

Comment:
should we name this file to `dbr_mock_test.go` to ignore this file to generate the test file and ignore the test coverage?

Reply:
If we should replace mock file to `internal/test/mock`, I'll adopt next PR(mysql test part2).
Is it OK?

Reply:
ok.
If we do so, do we need to update coding guideline?

Reply:
I think if we use mock for test, we should write handling test with test at test section in coding guideline

Reply:
lets discuss it in different place :)